docs/learn-faq.md

```diff
@@ -6,80 +6,327 @@ sidebar_label: Frequently Asked Questions (FAQs)
+security as Polkadot. The closest contenders are operating with around 150
+validators, while Kusama is running with 180 securely as of April 2020.
 
-It is important to note that the cost of transferring KSM or DOT tokens is dynamic. Currently, the minimum cost of transferring KSM is 0.01 KSM (the base fee), although this can be changed via governance. However, actual transaction fees will vary based on a variety of factors. Specifically, fee calculation follows the following formula:
+Additionally, other projects sometimes have a different definition of _validator_
+that approximates more closely to remote signing keys without the full
+operation of a validating node. On Polkadot, each validator is running their own
+validating node and performing full verification of the Relay Chain, voting on
+finality, producing blocks in their decided slots, and verifying parachain state
+transitions. Other projects may consider validators and "validating nodes" as
+separate entities.
 
-```
-base_fee + (tx_length * length_fee) + WeightToFee(weight)
-```
+## Relay Chain
 
-Please see the (Substrate page on fee calculation)[https://substrate.dev/docs/en/next/development/module/fees#fee-calculation] for more detailed information.
+### What is the expected block time on the Relay Chain?
 
-### What is the minimum amount of KSM I can have in my account?
+The Kusama network, an early and unaudited release of the Polkadot code is
+currently operating at a rate of one block every six seconds. 
 
-It is recommended to always ensure that you keep at least 0.1 KSM in your account in order to avoid the reaping threshold of 0.01 KSM.  If you have less than 0.01 KSM in your account, that account will be "reaped" - it will be removed and no longer occupy space on the chain. In other words, no accounts are allowed on-chain with an account balance of less than 0.01 KSM.   This is a dust prevention measure, in order to ensure that the chain is not full of accounts with minuscule amounts of KSM taking up space. Since the blockchain is copied to every person running a full node, any savings of space provides dramatic benefits in terms of scalability.
+It is expected that Polkadot will also target its block production for producing
+a block every six seconds. However, it is still subject to change. It may go
+as low as two to three seconds after optimizations, or it may potentially
+increase in order to handle the capacity of the parachain networking in a live
+environment.
 
-### Can I buy or transfer DOT tokens?
+### Does Polkadot have smart contracts?
 
-Testnet DOT and KSM tokens are freely available from a variety of sources - see the [DOT page](learn-DOT) for details.
+No - and yes. The Polkadot Relay Chain does not implement smart contracts
+natively. The reason for not having smart contracts on the Relay Chain is 
+party of the design philosophy for Polkadot that dictates it should be the 
```

Comment:
```suggestion
party of the design philosophy for Polkadot that dictates that the Relay Chain should be the 
```

The "it" was a little confusing - it could have referred to "Polkadot" itself since that was the last noun specified beforehand.